src/postgraphile/http/createPostGraphileHttpRequestHandler.ts

```diff
@@ -1,30 +1,49 @@
 /* eslint-disable */// Because we use tslint
+/* tslint:disable:no-any no-var-requires */
```

Comment:
and here's the real only significant offender of this PR.  to get the build to auto-populate typings, `allowJs` must be set to false.  this was the lone `.js` in the lib.

this file could use have used some deep, deep tidy before i put my fingers on it.  do to the size and complexity of it, rather than put proper types on everything in the true TS spirit, I declared _many_ of the internals as `any`.  this was to prevent any major refactor or create _even more change_ than we already want for this PR.